### PR TITLE
Octicon contribution for RemoveFilter

### PR DESCRIPTION
**Pull request by UnicodeRogue**
Closes https://github.com/github/memex/issues/15800

Connected to this Epic: https://github.com/github/primer/issues/2396

I'm working through the contribution guidelines posted here: https://github.com/primer/octicons/blob/6cbce08cb1fde5e316a9a079c2938b7bc80e50f6/CONTRIBUTING.md 

Which asks me to make sure to cover: 

> where the icon will be used: 

In the [SelectPanel](https://primer.style/design/components/selectpanel), to indicate removing a filter

> any relevant timeline information: 

I'm not sure if this refers to the timeline-to-shipping, but I do want to note that as I was adding this icon, I noticed most other icons had their styles written out as a single line (like filter-16.svg, as one example) and I wanted to check if that needed to be the case for new icons as well

### CODE REVIEW

**Comment by gr2m** [comment]
Great pull request and description 👍🏼 Based on other icons it seems you'll have to add a 24px variation as well?

> No Changeset found

try to run `npx changeset` and follow instructions

**Comment by UnicodeRogue** [comment]
> Great pull request and description 👍🏼 Based on other icons it seems you'll have to add a 24px variation as well?

ah good call! Added ✔️  and changeset command run ✔️

**Comment by gr2m** [comment]
sounds like you need to update snapshots:

https://github.com/primer/octicons/actions/runs/5904633153/job/16017173213?pr=976#step:8:9

try running `npx jest -u`

**Comment by UnicodeRogue** [comment]
note to say I'm continuing to troubleshoot the failing test 🚧

**Comment by UnicodeRogue** [comment]
running `npx jest -u` didn't resolve things- 

currently running `yarn test -u` to see how far I can get-

**Comment by UnicodeRogue** [comment]
Output of `npx jest -u`

Screenshot 2023-08-22 at 2 25 18 PM

**Comment by UnicodeRogue** [comment]
Output of running `yarn test -u`

Screenshot 2023-08-22 at 2 26 34 PM

File: icons/filter-remove-24.svg
Comment (gr2m) [inline icons/filter-remove-24.svg:1]: I think the 24px SVG needs to be updated use up the full space. I'm not sure if there is an easy hack to upscale a 16px version? I'm sure the @primer/octicons-reviewers team will be able to help to give some pointers 🙏🏼

Comment (UnicodeRogue) [reply icons/filter-remove-24.svg:1]: Sounds good, I'm checking with Design!

**Comment by gavinmn** [comment]
@UnicodeRogue Is there a related Octicons issue for this? I'm trying to understand if this is something the team looked at with you or if this is a new request altogether that we should take a look at in a working session.

**Comment by gr2m** [comment]
It's a new request, the icon was initially need for https://github.com/github/memex/issues/15800 (private GitHub repository), but will also be needed for the new SelectPanel: https://github.com/github/primer/issues/2396

**Review by gavinmn — APPROVED** [review]

**Review by gr2m — APPROVED** [review]
thank you @CameronFoxly - it looks great!

**Comment by gr2m** [comment]
@primer/octicons-reviewers can we get this pull request merged an released to conclude the work?

**Pull request merged**